src/engine/MaxConstraint.cpp

```diff
@@ -203,8 +205,8 @@ void MaxConstraint::notifyUpperBound( unsigned variable, double value )
 void MaxConstraint::getEntailedTightenings( List<Tightening> &tightenings ) const
 {
     // Lower and upper bounds for the f variable
-    double fLB = _lowerBounds.exists( _f ) ? _lowerBounds.get( _f ) : FloatUtils::negativeInfinity();
-    double fUB = _upperBounds.exists( _f ) ? _upperBounds.get( _f ) : FloatUtils::infinity();
+    double fLB = _lowerBounds.exists( _f ) ? _lowerBounds.get( _f ) : FloatUtils::negativeInfinity(); // todo make max(-inf, _maxValueOfEliminated) ?
+    double fUB = _upperBounds.exists( _f ) ? _upperBounds.get( _f ) : FloatUtils::infinity(); // todo make max(-inf, _maxValueOfEliminated)?
 
```

Comment:
the lower bound can indeed be initialized to maxValueOfEliminated, if it exists. For the upper bound, I don't think you can initialize to anything except infinity. Remove comments

Reply:
ok, in case _lowerBounds.exists( _f ) is "false" - I'll put maxValueOfEliminated, which is anyhow initialized to infinity.